_posts/2016/2016-08-01-praktikumsbericht-jb.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Forschungspraktikum in Norwegen
+categories: erfahrungsbericht
+author: Henning
```

Comment:
Wieso ist Autor Henning wenn der Artikel in 1. Person von Jaqueline geschrieben wurde?

Reply:
Berechtigter Einwand, habe ich aktualisiert